src/sql/base/browser/ui/table/plugins/buttonColumn.plugin.ts

```diff
@@ -83,6 +91,17 @@ export class ButtonColumn<T extends Slick.SlickData> implements Slick.Plugin<T>
 		}
 	}
 
+	// This call is to handle reactive changes in check box UI
```

Comment:
I don't understand why do we need to handle checkbox related logic in a button plugin, is this really necessary?

Reply:
not for checkbox, it's copy paste problem. but the code was needed after registering the plugin. the same code was in checkbox plugin

---
In reply to: [446433171](https://github.com/microsoft/azuredatastudio/pull/10918#discussion_r446433171) [](ancestors = 446433171)